MultiThreading.Task1.100Tasks/Program.cs

```diff
@@ -25,12 +26,18 @@ static void Main(string[] args)
         Console.ReadLine();
     }
 
-    static void HundredTasks()
+    private static void HundredTasks()
     {
-        // feel free to add your code here
+        Parallel.For(0, TaskAmount, (taskNumber) =>
```

Comment:
"write a program that creates an array of 100 Tasks"
This should be a task array probably

Reply:
changed an approach